cypress/pageObjects/profile/assertions.ts

```diff
@@ -13,6 +14,32 @@ class ProfilePageAssertions {
     cy.get("ng-transclude ").find(".ng-binding").should("contain", counter);
     return this;
   }
+
+  checkUsernameInProfileURL(username: string) {
+    cy.location().should((location) => {
```

Comment:
cy.url().should(isContain ? "contain" : "not.contain", value);

Reply:
checkUrlContainsValue(value: string, isContain: boolean) {
        cy.url().should(isContain ? "contain" : "not.contain", value);
        return this;
    }

Reply:
Create it in a new assertions file 'sharedAssertions.ts' under pageObjects folder.